server/rpc_test.go

```diff
@@ -559,14 +559,14 @@ func StubBroadcastSessionsManager() *BroadcastSessionsManager {
 	}
 }
 
-func TestSelect(t *testing.T) {
+func TestSelectSession(t *testing.T) {
 	bsm := StubBroadcastSessionsManager()
-	expectedSess1 := bsm.sessList[1]
-	expectedSess2 := bsm.sessList[0]
 
 	// assert that initial lengths are as expected
 	assert := assert.New(t)
 	assert.Len(bsm.sessList, 2)
+	expectedSess1 := bsm.sessList[1]
+	expectedSess2 := bsm.sessList[0]
```

Comment:
Moved this because if the session list happens to be less than length 2 then we'll segfault before the assert.Len